lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -4094,6 +4095,7 @@ ParseResult FIRCircuitParser::skipToModuleEnd(unsigned indent) {
     case FIRToken::kw_module:
     case FIRToken::kw_extmodule:
     case FIRToken::kw_intmodule:
+    case FIRToken::kw_class:
```

Comment:
hyper-nit: Should we alpha-sort these?  The `parseFOO` methods appear to be organized that way.

Reply:
alphabetized 👍